File: pset1/Cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void){
    float change = 0;
    int coins = 0;

    do{
        change = get_float("change owed :");
    }while(change < 0);

    int cents = round(change * 100);

    do{
        if(cents >= 25){
            cents -= 25;
            coins++;
        }else if(cents >= 10 & cents < 25){
            cents -= 10;
            coins++;
        }else if(cents >= 5 & cents < 10){
            cents -= 5;
            coins++;
        }else{
            cents -= 1;
            coins++;
        }
    }while(cents > 1);

    printf("%i coins \n", coins);
}

File: pset6/cash/cash.py
import cs50

change = 0
coins = 0

change = cs50.get_float("change owed : ")

cents  = change * 100

while True:
    if cents >= 25:
        cents -= 25;
        coins += 1;

    elif cents >= 10 and cents < 25:
        cents -= 10;
        coins += 1;

    elif cents >= 5 and cents < 10:
        cents -= 5;
        coins += 1;

    else:
        cents -= 1;

    if cents == 0:
        break
print(f"{coins} coins")